.github/workflows/stub.yml

```diff
@@ -0,0 +1,25 @@
+name: Check stub
```

Comment:
I would put all the content in the `ci` file.  WDYT?

Reply:
Done